operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -878,6 +878,32 @@ public static X509Certificate cert(Secret secret, String key)  {
         }
     }
 
+    /**
+     * Returns concatenated string of all public keys (all .crt records) from a secret
+     *
+     * @param secret    Kubernetes Secret with certificates
+     *
+     * @return          String secrets
+     */
+    public static String certsToString(Secret secret)  {
```

Comment:
I wonder if this should really be in the `Ca` class. It is not really related to the CA it self, or? Have you considered putting this into some Utils class?